src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Nadahar You still win a little with this command: ;-)
ffmpeg64.exe -y -hwaccel auto -i jellyfish-60-mbps-hd-hevc-10bit.mkv -c:v h264_mfx -tune fastdecode -preset superfast -crf 22 -c:a aac -b:a 320k -ar 48000 -pix_fmt yuv420p -f ismv -movflags isml+frag_keyframe hevc_test1.mp4 -benchmark